Packs/BPA/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,3 @@
+#### Playbooks
+##### Comprehensive PAN-OS Best Practice Assessment
+- Documentation and metadata improvements.
```

Comment:
Why not mentioning the fixed typo? so whoever reported it will know it got fixed